app/models/host/base.rb

```diff
@@ -246,6 +246,10 @@ def overwrite=(value)
     end
 
     def primary_interface
+      # Returns a dummy object if host is unmanaged and without interfaces.
+      # Unmanaged hosts still have the interface requirement due to delegations and certain validations.
+      # A refactoring that allows unmanaged hosts to exist without interfaces would be welcome.
+      return Nic::Base.create(:host => self, :primary => true) unless new_record? || managed || interfaces.present?
```

Comment:
We pass :host => self and :primary => true so that NameSynchronizer sets the name of the host properly.

....rant.....

Yes.

I also think it couldn't be dirtier but 1.8 is near and we won't have another RC for testing.
=